src/metabase/sync_database/sync.clj

```diff
@@ -143,16 +143,24 @@
       (catch Throwable t
         (log/error (u/format-color 'red "Unexpected error syncing table") t)))))
 
-(def ^:private ^:const blacklisted-table-names
-  "Names of Tables to skip syncing. These should be lowercased, as we'll compare them with lowercased names of `raw-tables` below.
-   These are Tables that are known to not contain useful data, such as migration or web framework internal tables."
-  #{"_metabase_metadata"
-    ;; Django
+(def ^:private ^:const crufty-table-names
+  "Names of Tables that should automatically given the `visibility-type` of `:cruft`.
+   This means they are automatically hidden to users (but can be unhidden in the admin panel).
+   These `Tables` are known to not contain useful data, such as migration or web framework internal tables."
+  #{;; Django
+    "auth_group"
+    "auth_group_permissions"
+    "auth_permission"
     "django_admin_log"
     "django_content_type"
+    "django_migrations"
     "django_session"
     "django_site"
     "south_migrationhistory"
+    "user_groups"
+    "user_user_permissions"
+    ;; Rails / Active Record
+    "schema_migrations"
```

Comment:
Rails 5 brought a new table called `ar_internal_metadata` ([reference](https://blog.bigbinary.com/2016/06/07/rails-5-prevents-destructive-action-on-production-db.html)). But the addition of new internal tables has been a very rare event in Rails development so far.

Reply:
@aCandidMind if you want to submit a PR to add it we'd love to have you as a contributor 😄  As a heads up we moved the lists to `src/metabase/sync/sync_metadata/tables.clj`

Reply:
All I know about Clojure is from conferences and I have never run metabase. Just landed here through discussions about this new table in Rails. I guess I should leave this to others.

Reply:
Thanks for the heads up regardless. We appreciate you chiming in!